src/agent/communicator/include/https_verifier_mac.hpp

```diff
@@ -0,0 +1,90 @@
+#include <icert_store_utils_mac.hpp>
```

Comment:
```suggestion
#pragma once

#include <icert_store_utils_mac.hpp>
```
Check all `.hpp` files.

Reply:
Done!